src/main/java/io/lakefs/routerfs/PathMapper.java

```diff
@@ -116,11 +135,12 @@ private List<MappingConfig> parseMappingConfig(Configuration conf) throws Invali
      */
     private void logLoadedMappings() {
         LOG.debug("pathMappings: ");
```

Comment:
nit:
```suggestion
        LOG.debug("pathMappings:");
```

Also this header might be easier to understand if the next lines are all indented.